### PR TITLE
changed GenHub to Genome or fidibus for logging messages

### PR DESCRIPTION
**Pull request by vpbrendel**

### CODE REVIEW

**Comment by vpbrendel** [comment]
Note the the INSTALL document needs  to be updated once the Singularity recipe has been accepted by this pull request - then the container will be built and posted on the hub.

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# AEGeAn Installation and Setup
+
+## Installation as a singularity container [![https://www.singularity-hub.org/static/img/hosted-singularity--hub-%23e32929.svg](https://www.singularity-hub.org/static/img/hosted-singularity--hub-%23e32929.svg)](https://singularity-hub.org/collections/3301)
```
Comment (standage) [inline INSTALL.md]: I'm a big fan of the container approach! My only concern is that the AEGeAn documentation doesn't become fragmented. I've been using ReadTheDocs to host [AEGeAn's documentation](https://aegean.readthedocs.io/en/stable/) for a while—it detects when the master branch has been updated or a new release has been published, and rebuilds the documentation from the `docs/` directory automatically. We don't want redundant or conflicting installation procedures in different places.

I think the installation instructions you've provided here are a great improvement over the install procedures we've used in the past. So the question is whether it makes sense to move these new instructions into the `docs/` directory? Or move the other contents of the `docs/` directory  somewhere else?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+## Installation
+Please find detailed installation instructions and options in the
+[INSTALL](./INSTALL.md) document.
+
+
+## FAQ
+
+Please see
+[our wiki FAQ pages](https://github.com/BrendelGroup/AEGeAn/wiki/FAQ)
+for usage examples and suggestions.
```
Comment (standage) [inline README.md]: Same concern as before: I have no problem using the wiki if that's the way we want to go, but we should avoid having documentation split across different places. The `docs/` directory already has several pages explaining what programs are available, how to run them, GFF3 conventions, and so on.

Comment (standage) [reply README.md]: By the way, I forgot that I had ever put any content in the wiki. The notes on automated testing are a distant memory: probably a checklist I had planned on using to keep track of features needing testing. In any case, I never ended up using it and I'll be happy to delete it after six years of neglect.

Comment (vpbrendel) [reply README.md]: The idea behind the reorganization is to simplify matters and make maintenance easy. The readthedocs site is rather elaborate, and I'd be happy for you to maintain such document. But for someone wanting to install the code and run it, it's much better to have everything a click away from the home page, as is common with many github projects. The FAQ is just another place, easily accessible, where odd cases that come up can be documented.
Thus, I would suggest to simplify documentation, license, installation as per usual group style, but then for you to maintain a User's Manual as the readthedocs document, which can provide more detailed explanations on all the topics not covered in publications. Such an approach is also quite common, providing both the quick start right at hand and more details elsewhere.

Comment (standage) [reply README.md]: If we were to move the content of your new `INSTALL.md` file to the existing `docs/install.rst`, we could add a link to the rendered page to the README. That would certainly satisfy the "one click away" convenience, and the maintenance burden shouldn't be any different than for a markdown file in the root directory.

But taking a look at other recently updated git repos in the `BrendelGroup` space, I see what you mean by consistent group style. I'll treat the ReadTheDocs site as a user manual, and update the corresponding install instructions to point to the new `INSTALL.md` file as the endorsed installation procedure.

**Pull request merged**